test/fixtures/slack_identifiers.yml

```diff
@@ -0,0 +1,11 @@
+deployer:
+  identifier: 'U1234567890'
```

Comment:
maybe `ident-deployer` or something like that to make it easier to decypher what is going on in tests ?